src/clp_ffi_py/encoder/encoding_methods.cpp

```diff
@@ -0,0 +1,137 @@
+#include <clp_ffi_py/ErrorMessage.hpp>
+#include <clp_ffi_py/Python.hpp>
+#include <clp_ffi_py/encoder/encoding_methods.hpp>
+
+#include <clp/components/core/src/ffi/encoding_methods.hpp>
+#include <clp/components/core/src/ffi/ir_stream/encoding_methods.hpp>
+#include <clp/components/core/src/type_utils.hpp>
```

Comment:
```suggestion
#include <clp_ffi_py/encoder/encoding_methods.hpp>

#include <clp/components/core/src/ffi/encoding_methods.hpp>
#include <clp/components/core/src/ffi/ir_stream/encoding_methods.hpp>
#include <clp/components/core/src/type_utils.hpp>

#include <clp_ffi_py/ErrorMessage.hpp>
```

* The style-guide dictates that the first include should be the corresponding `.hpp` for this `.cpp`, followed by third-party includes (in this case, clp), followed by project-specific includes.
* We should enforce that `Python.hpp` is included first, which in this case is easiest to do by simply deleting it from this file and relying on the include from `encoding_methods.hpp`